lib/sidekiq/grouping/version.rb

```diff
@@ -1,5 +1,5 @@
 module Sidekiq
   module Grouping
-    VERSION = '1.0.2'
+    VERSION = '1.0.3'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.